### ISSUE TITLE
Refactor PredefinedShelfUtils to have static methods

### ISSUE THREAD

**Issue by knjk04** [Project-Books/book-project#436]
Currently, `PredefinedShelfUtils` has non-static methods and needs to be instantiated. Refactor this so it cannot be instantiated and all of its methods are static

---

If you need anything, feel free to get in touch with us over [Slack](https://join.slack.com/t/teambookproject/shared_invite/zt-h6vx3n8l-KN8QnO50r7QWZHgHFnFdPw)

**Comment by daviduster** [comment]
Hi! @knjk04, I would like to help on this

**Comment by PirtiXcode** [comment]
Hey I want to contribute on this issue.

**Comment by pnpninja** [comment]
May I?

**Comment by pnpninja** [comment]
When you mean all of its methods - what about the methods that need PredefinedShelfService that gets defined in the constructor?

**Comment by pnpninja** [comment]
I can autowire the PredefinedShelfService, make it static and then make all methods static. Is that a good idea?

**Comment by pnpninja** [comment]
If you say yes to the last comment, I have already made those changes

**Comment by knjk04** [comment]
@daviduster Sure, I'll assign you to the issue. Thanks!

**Comment by knjk04** [comment]
@PirtiXcode Sorry, David asked before you. I also noticed that you have already requested to be assigned to an issue. Please note that, as per our contributing guidelines, we only assign one issue at a time to new contributors

**Comment by knjk04** [comment]
@pnpninja Sorry, I have already assigned the issue. Please note that, as per our contributing guidelines, we expect contributors to wait to be assigned before working on the issue. I know you already created a pull request, but as David asked first, I'm afraid we won't be able to accept your pull request for this issue.

If you would like to request to be assigned to a different issue that is unassigned and unblocked, please feel free.

**Comment by pnpninja** [comment]
@knjk04 please assign me to something unassigned and unblocked then..

**Comment by pnpninja** [comment]
Also @daviduster - feel free to use my fork and commit branch to get this issue solved

**Comment by knjk04** [comment]
@pnpninja I cannot assign you the issues you requested to be assigned to. If there's another issue that you would like to work on, please request to be assigned to it

**Comment by knjk04** [comment]
Removing the 'hacktoberfest' label as we may not be able to get this merged in before the 31st of October, but this is still up for grabs

**Comment by kndarp** [comment]
Hi @knjk04 👋🏽  
I'm looking forward to my first contribution to this repo. If this issue is still available, I'd like to be assigned to this issue.

**Comment by knjk04** [comment]
Hi @kndarp,

Unfortunately, this issue is blocked. I'll let you know if another good first issue comes up, though!

**Comment by knjk04** [comment]
This file no longer exists as we have moved it to the predefined shelf service
